### ISSUE TITLE
Suggestion: Change Toad's and Toadette's recolorability.

### ISSUE THREAD

**Issue by Brob2nd** [coop-deluxe/sm64coopdx#396]
I don't know if this counts as a coopdx suggestion, or as a mod suggestion, but i'm still suggesting it.

First: make Toad's recolorability on his circles on the mushroom hat be Emblem instead of Cap, at least for me, this makes more sense than making it Cap.
Also, change the other recoloraeable part on his mushroom hat be Cap instead of Gloves, just like Toadette, this would make Gloves the only unused recoloreable part on Toad, but i mean, Emblem's also unused, and it makes more sense to me to make it the way i'm saying it should be.

Second  (Here's when the mod related part starts):  Now, second, making that change would also require to Change Toadette's recolorability, for some reason, her circles on the mushroom hat are Gloves, instead of Cap, so, we can change it easily to Emblem, now, the recolorability on Toadette is weird, because when changing the color on Hat, both the Hat and the middle shirt changes, so, how you fix this? It would be required to also apply the recolorability changes from Toad's hat to Toadette, then, i would suggest to make the Middle Shirt have the overalls recolorability instead of the white line in her, then, change the white's line recolorability to Gloves, and problem solved.

TLDR: The suggestion is basically, changing Toad's and Toadette's recolorability to what's shown in the image, since it makes more sense to me, thanks for reading this, sm64coopdx devs.

![Captura de pantalla (917)](https://github.com/user-attachments/assets/b0a8503b-e964-4c1d-a65d-ecf0740934a3)
![Captura de pantalla (916)](https://github.com/user-attachments/assets/9b9527b9-d7ff-4552-b171-81f363a71c84)
